### PR TITLE
Users controller

### PR DESCRIPTION
**Pull request by Aisselaa**
Hey team,
I've created the user controller with edit controller and show

### CODE REVIEW

**Comment by Aisselaa** [comment]
Screenshot 2022-06-14 at 22 14 14

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -5,10 +5,10 @@ class ApplicationController < ActionController::Base
 
   def configure_permitted_parameters
     # For additional fields in app/views/devise/registrations/new.html.erb
-    devise_parameter_sanitizer.permit(:sign_up, keys: [:first_name, :last_name])
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:first_name, :last_name, :age, :biography, :type_of_dwelling, :outdoor_space, :other_pets, :children, :hours_alone, :previous_experience ])
```
Comment (soojinhwang) [inline app/controllers/application_controller.rb:8]: Leave first_name and last name, but take out others

File: app/controllers/pages_controller.rb [hunk 2]
```diff
@@ -4,5 +4,11 @@ class PagesController < ApplicationController
   def home
   end
 
+<<<<<<< HEAD
```
Comment (soojinhwang) [inline app/controllers/pages_controller.rb:7]: remove Head

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  end
+
+  private
+
+  def set_user
+    @user = current_user
+  end
+
+  def user_params
+    params.require(:user).permit( :email, :password, :first_name, :last_name, :age, :biography, :type_of_dwelling, :outdoor_space, :other_pets, :children, :hours_alone, :previous_experience )
```
Comment (soojinhwang) [inline app/controllers/users_controller.rb:27]: add photo in the permit as well (look at games)

File: app/models/user.rb [hunk 4]
```diff
@@ -10,19 +10,19 @@ class User < ApplicationRecord
   has_one_attached :photo
 
   validates :email, presence: true
-  validates :password, presence: true
+  #validates :password, presence: true
```
Comment (soojinhwang) [inline app/models/user.rb:13]: uncomment this

File: app/models/user.rb [hunk 5]
```diff
@@ -10,19 +10,19 @@ class User < ApplicationRecord
...
-  validates :type_of_dwelling, inclusion: { in: %w(House Flat Farm),
-                                            message: "%{value} is not a valid type of dwelling" },
-                               unless: ->(x) { x.type_of_dwelling.blank? }
-  validates :outdoor_space, inclusion: { in: [true, false],
-                                          message: "%{value} is not a valid option" },
-                            unless: ->(x) { x.outdoor_space.blank? }
-  validates  :hours_alone, numericality: { only_integer: true, greater_than_or_equal_to: 0, less_than_or_equal_to: 24 },
-                           unless: ->(x) { x.hours_alone.blank? }
-  validates  :previous_experience, length: { minimum: 10 }, unless: ->(x) { x.previous_experience.blank? }
+  # validates :age, numericality: { only_integer: true }, unless: ->(x) { x.age.blank? }
```
Comment (soojinhwang) [inline app/models/user.rb:17]: uncomment the validations

**Pull request merged**